nservicebus/messaging/send-a-message.md

```diff
@@ -1,43 +1,54 @@
+
+
+## Inside the pipeline
+
+Messages are sent from within the message processing pipeline when they are sent from a message handler, a saga or some advanced extensibility points. This is code triggered by an incoming message and all send operations are linked to that incoming message. This allows transactions to rollback already sent messages when the message fails at some later stage in the pipeline.
+
+Here's how you send a simple message from a message handler:
 
 snippet:SendFromHandler
 
-In both cases you can use an interface rather than a concrete class for a message:
+It's also possible to use an interface rather than a concrete class for a message:
 
 snippet:BasicSendInterface
 
+Note: In NServiceBus Version 5 and below, the operations are available on the `IBus` which can be accessed using constructor or property injection. From version 6 `IMessageHandlerContext` is passed in directly to provide access to bus operations.
+
 
-## Immediate Dispatch
+### Immediate Dispatch
 
 While its usually best to let NServiceBus [handle exceptions for you](/nservicebus/errors), there are some scenarios where you want to send messages out even though the incoming message is rolled back. One example would be sending a reply notifying that there was an issue processing the message.
 
-In order to request immediate dispatch you can use the following syntax.
+In order to request immediate dispatch use the following syntax.
 
 snippet:RequestImmediateDispatch
 
 NOTE: By specifying immediate dispatch, outgoing messages are not [batched](/nservicebus/messaging/batched-dispatch.md) or enlisted in the current receive transaction even if the transport has support for it.
 
-
-### Suppressing the transaction scope
-
-Version 6 and below allows you to suppress the ambient transaction in order to have the outgoing message sent immediately.
+bVersion 6 and below also allows suppressing the amient transaction in order to have the outgoing message sent immediately.
 
 snippet:RequestImmediateDispatchUsingScope
 
-The issue with this approach is that it only works for transports that enlists the receive operation in a transaction scope. Currently this would be MSMQ and SqlServer in DTC mode. Should you use any other transport or disable the DTC this no longer works and the outgoing message might be rolled back together with the incoming message.
-
-For this reason we've decided to deprecate this method and recommend users switch to the explicit API mentioned above.
\ No newline at end of file
+WARNING: This method is being deprecated in favor of the explicit immediate dispatch in Version 6. It only works for MSMQ or SqlServer running in DTC mode. Should any other transport be used, or DTC be disabled, the outgoing message may be rolled back together with the incoming message.
```

Comment:
The warning is confusing. It's in the section for Immediate Dispatch.
Rather than say, "This method", perhaps state the actual method name that is being deprecated in favor of the new method name? Also, isn't the immediate dispatch a v6 concept?